src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -143,16 +145,25 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
       template = <div>Unknown Template: ${(venue as AnyVenue).template}</div>;
   }
 
-  // @debt remove backButton from Navbar
+  const parentVenueId = venue?.parentId ?? "";
+
+  const backToParentVenue = useCallback(() => {
+    enterVenue(parentVenueId, { customOpenRelativeUrl: history.push });
+  }, [history.push, parentVenueId]);
```

Comment:
```suggestion
  const backToParentVenue = useCallback(() => {
    if (!parentVenueId) return
    
    enterVenue(parentVenueId, { customOpenRelativeUrl: history.push });
  }, [history.push, parentVenueId]);
```

Reply:
Instead of `history.push`, it might be good to extract.rename the function we're using in the same way that the existing code that is being removed did it (eg. `openUrlUsingRouter` in the `NavBar`)

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1492/commits/516bcef90616e538cbf0a8609a96d134bfbf4fcc